package-lock.json

```diff
@@ -3437,9 +3437,9 @@
       "dev": true
```

Comment:
If there is no package update, I will suggest not commit package-lock file, to reduce the conflicts.